content/rancher/v2.x/en/operation/ingress/_index.md

```diff
@@ -5,3 +5,62 @@ draft: true
 ---
 
 ---
+
+**Introduction**
+
```

Comment:
By default, applications deployed in a K8s cluster are only accessible from the overlay network within the cluster. To expose an application to clients outside of the cluster, a mechanism must be in place to route external requests to the pods providing the application.

Reply:
Ok